Source/Resources/Content/MSGraph/DeviceManagement/WindowsAutopilotDeploymentProfiles/Baseline%3A Self Deploying Profile.json

```diff
@@ -1,30 +1,40 @@
-  "assignments": [
-    {
-      "target": {
-        "@odata.type": "#microsoft.graph.groupAssignmentTarget",
-        "groupId": "${urn:resource:MSGraph:Groups/Baseline - Autopilot Devices - Self Deploying?$id}"
-      }
-    }
-  ],
-  "description": "| What does this do? | Creates an Intune Autopilot profile for enrolling machines using the self deploying method, which enables a device to be enrolled into your environment with little to no user interaction. Self deployment mode comes with restrictions including that the device must have TPM 2.0, and it is not supported on virtual machines even if they have a virtual TPM. Devices in the Azure AD group \"Baseline Autopilot Devices Self Deploying\" will be assigned this profile. |\n| Why should you use this? | This is most useful for devices that will be shared or used as a kiosk. If a device is going to be used by a single user it is best to use the user driven method. |\n| What is the end user impact? | User devices assigned to this profile can be configured using the self deployment method. |",
-  "deviceNameTemplate": "",
-  "deviceType": "windowsPc",
-  "enableWhiteGlove": false,
-  "extractHardwareHash": true,
-  "language": "en-US",
-  "outOfBoxExperienceSettings": {
-    "deviceUsageType": "shared",
-    "hideEscapeLink": true,
-    "hideEULA": true,
-    "hidePrivacySettings": true,
-    "skipKeyboardSelectionPage": true,
-    "userType": "standard"
-  },
-  "roleScopeTagIds": [
-    "${urn:resource:MSGraph:DeviceManagement:RoleScopeTags/Default?$id}"
-  ]
-}
+{
+  "@odata.type": "#microsoft.graph.azureADWindowsAutopilotDeploymentProfile",
+  "$description": "| What does this do? | Creates an Intune Autopilot profile for enrolling machines using the self deploying method, which enables a device to be enrolled into your environment with little to no user interaction. Self deployment mode comes with restrictions including that the device must have TPM 2.0, and it is not supported on virtual machines even if they have a virtual TPM. Devices in the Azure AD group \"Baseline Autopilot Devices Self Deploying\" will be assigned this profile. |\n| Why should you use this? | This is most useful for devices that will be shared or used as a kiosk. If a device is going to be used by a single user it is best to use the user driven method. |\n| What is the end user impact? | User devices assigned to this profile can be configured using the self deployment method. |",
+  "$name": "Baseline: Self Deploying Profile",
+  "assignments": [
+    {
+      "target": {
+        "@odata.type": "#microsoft.graph.groupAssignmentTarget",
+        "groupId": "${urn:resource:MSGraph:Groups/Baseline - Autopilot Devices - Self Deploying?$id}"
+      }
+    }
+  ],
+  "deviceNameTemplate": "",
+  "deviceType": "windowsPc",
+  "enableWhiteGlove": false,
+  "extractHardwareHash": true,
+  "hardwareHashExtractionEnabled": true,
+  "language": "en-US",
+  "locale": "en-US",
+  "outOfBoxExperienceSetting": {
```

Comment:
is this the correct property name?

Reply:
Nevermind, I see this is a change to a different namespace from the one with the LocalAdmins property change